examples/solvers/kaczmarz_tomography.py

```diff
@@ -0,0 +1,74 @@
+"""Tomography using the `kaczmarz` solver.
+
+Solves the inverse problem
+
+    A(x) = g
+
+Where ``A`` is a fan (cone) beam forward projector, ``x`` the result and
+ ``g`` is given data.
+
+In order to solve this using `kaczmarz`'s method, the operator is split into
+several sub-operators (each representing a subset of the angles and detector
+points). This allows a faster solution.
+"""
+
+import odl
+
+
+# --- Set up the forward operator (ray transform) --- #
+
+
+# Reconstruction space: discretized functions on the rectangle
+# [-20, 20]^2 with 300 samples per dimension.
+space = odl.uniform_discr(
+    min_pt=[-20, -20], max_pt=[20, 20], shape=[300, 300], dtype='float32')
+
+# Make a fan (cone) beam geometry with flat detector
+geometry = odl.tomo.cone_beam_geometry(space,
+                                       src_radius=40, det_radius=40,
+                                       num_angles=360, det_shape=360)
+
```

Comment:
This only works with astra? I receive the error message

`TypeError: 'skimage' backend only supports 2d parallel geometries
`

Reply:
I made some modifications so we should be able to use this with parallel beam instead. I'll change that.